.dockerignore

```diff
@@ -0,0 +1,5 @@
+Dockerfile
```

Comment:
If a Dockerfile change affects the image, then the image will get rebuilt by the successive `docker build` anyway. The motivation here was not to trigger unneeded builds in the case when changes to Dockerfile do not affect the image content, eg formatting adjustments. Such changes would normally invalidate a cached `COPY . .` layer and trigger a rebuild of successive layers, which can take some time if they happen to involve `go get` or `bundle install`.